src/amo/components/AddonQRCodeLink/index.js

```diff
@@ -0,0 +1,114 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import config from 'config';
+
+import { CLIENT_APP_ANDROID } from 'amo/constants';
+import AddonQRCode from 'amo/components/AddonQRCode';
+import Button from 'amo/components/Button';
+import OverlayCard from 'amo/components/OverlayCard';
+import translate from 'amo/i18n/translate';
+import type { AppState } from 'amo/store';
+import type { AddonType } from 'amo/types/addons';
+import type { ElementEvent, HTMLElementEventHandler } from 'amo/types/dom';
+import type { I18nType } from 'amo/types/i18n';
+
+import './styles.scss';
+
+type Props = {|
+  addon: AddonType,
+|};
+
+export type DeafultProps = {|
```

Comment:
```suggestion
export type DefaultProps = {|
```

Reply:
That code was copied from another module. When I do a search through our codebase for `DeafultProps` I see it in 8 places. 😢 

Do you think I should fix all 8 of those in this patch, or file a follow-up to do all of them separately?

Reply:
> Do you think I should fix all 8 of those in this patch, or file a follow-up to do all of them separately?

I'd fix this one here and submit a different PR to fix the other occurrences in the codebase.

Reply:
FTR, I corrected this in this component and `AddonQRCode` in this patch, but didn't think I should touch any of the other files in this patch. I will file a follow-up.